types/encoding.go

```diff
@@ -115,6 +115,35 @@ func (c *Currency) UnmarshalSia(r io.Reader) error {
 	return nil
 }
 
+// HumanString prints the Currency using human readable units. The unit used
+// will be the largest unit that results in a value greater than 1. The value is
+// rounded to 4 significant digits.
+func (c Currency) HumanString() string {
```

Comment:
This should probably have unit tests to verify its correctness given various currency amounts (just copy the `TestCurrencyUnits` test from siac)

Siac should probably be updated to use this method also